### DISCUSSION TITLE
Can't run the MacYTDL on macOS 11 and 12

### DISCUSSION THREAD

**Discussion by ghost** [section83/MacYTDL#38] [Q&A]
Whenever I start MacYTDL on macOS 10.15 or 11, the app will say
 "There was a problem with downloading FFmpeg. Perhaps you are not connected to the internet or the server is currently not available. MacYTDL can't run and will have to quit. When you are sure you are connected to the internet, re-open MacYTDL. MacYTDL, will then try to install FFmpeg."

**Comment by section83** [comment]
Sorry you are having this problem.  That message is issued when MacYTDL can't get the web page from which it downloads copies of FFmpeg.  Sometimes, that page is not available (e.g. for updates) and so MacYTDL can't get the page – but that is rare.

Can you open this link in your web browser ?   https://evermeet.cx/pub/ffmpeg/. If you can't see that web page, maybe you have something blocking it (e.g. LittleSnitch).

Can you also try this:

- Open Terminal;
- Copy and paste this at the prompt:

curl  "https://evermeet.cx/pub/ffmpeg/" | textutil -stdin -stdout -format html -convert txt -encoding UTF-8

You should see the web page text in your Terminal window.  If you do, MacYTDL should be able to get the page too – try MacYTDL again.  If you get an error instead, please report back here what the error was.

If it still doesn't work you can manually download and install FFmpeg:

- Download FFmpeg from here: https://evermeet.cx/pub/ffmpeg/ffmpeg-4.4.zip
- Open the Zip file and move the FFmpeg executable to: usr/local/bin [you will need to provide admin credentials]
- Do the same for FFprobe: https://evermeet.cx/pub/ffprobe/ffprobe-4.4.zip

MacYTDL should then run normally.

Thanks.

Garry
